From a612da71491efd1e926b986741f30f2f2055b685 Mon Sep 17 00:00:00 2001
From: Brittany Wald <brittany.wald@joyent.com>
Date: Tue, 15 Aug 2017 21:29:59 +0000
Subject: [PATCH] MANTA-3371 triggers cannot be installed with any vnodes in
 read-only mode Reviewed by: Joshua M. Clulow <jmc@joyent.com> Reviewed by:
 Richard Bradley <richard.bradley@joyent.com> Approved by: Richard Bradley
 <richard.bradley@joyent.com>

---
 package.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package.json b/package.json
index a82fad1..490ff4a 100644
--- a/package.json
+++ b/package.json
@@ -33,7 +33,7 @@
         "mahi": "2.0.1",
         "marlin": "git+https://github.com/joyent/manta-marlin.git#master",
         "mime": "1.2.11",
-        "moray": "3.1.1",
+        "moray": "3.3.0",
         "once": "1.3.0",
         "restify": "2.6.3",
         "vasync": "^1.5.0",
-- 
2.21.0

